src/main/io/serial_msp.c

```diff
@@ -1436,6 +1439,14 @@ static bool processInCommand(void)
         }
 #endif
         break;
+    case MSP_SET_SERVO_LIMIT:
+#ifdef USE_SERVOS
```

Comment:
at first glace i can't see why this is needed since MSP_SERVO_CONF has also been updated and seems correct.